calculator.rb

```diff
@@ -0,0 +1,56 @@
+
+def calculator(op, num1, num2)
+
+  if op == "add" || op == "+"
```

Comment:
I like that you use a method here, but this method ends up being pretty big, especially when your operations are complex (like divide, where you need to test for 0). A good strategy here might be to split each operation out into its own method.